dist/i18n-react.js

```diff
@@ -199,17 +199,15 @@ var MDText = /** @class */ (function () {
             trans = (options && options.notFound !== undefined) ? options.notFound :
                 this.notFound !== undefined ? this.notFound :
                     key;
-            if (isFunction(trans)) {
-                trans = trans(key, options);
-            }
-            if (!isString(trans)) {
-                return trans;
-            }
         }
         if (isFunction(trans)) {
-            trans = trans(key, context);
+            var args = context === undefined ? options : context;
+            trans = trans(key, args);
```

Comment:
I don't understand why args are either options or context

Reply:
There are tests that test `notFound` function call with context (`translate(key, { context: context})`.  I guess a lot of users use this feature so I should not just switch `context` to `options`. I tried to merge `context` and `options` ( `{ ...options, ...context }`) but `context` can be just plain number or string and the merge doesn't work for these cases.
So I decided that if an user passed `context` - the user should receive translation with the `context`, otherwise `options` should be passed.